.github/workflows/audit.yml

```diff
@@ -12,15 +12,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Set up Go
```

Comment:
Can you also change the name of the action?

Reply:
Currently, it is `Go Nancy`.

Reply:
done